{
  "cve": "CVE-2018-7431",
  "mitre": {
    "cpes": [],
    "created": "2018-10-23T21:00:00+00:00",
    "description": "Directory traversal vulnerability in the Splunk Django App in Splunk Enterprise 6.0.x before 6.0.14, 6.1.x before 6.1.13, 6.2.x before 6.2.14, 6.3.x before 6.3.10, 6.4.x before 6.4.6, and 6.5.x before 6.5.3; and Splunk Light before 6.6.0 allows remote authenticated users to read arbitrary files via unspecified vectors.",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {},
      "cvssV3_1": {},
      "cvssV4_0": {}
    },
    "mitre_repo_path": "cves/2018/7xxx/CVE-2018-7431.json",
    "references": [
      "https://www.splunk.com/view/SP-CAAAP5T"
    ],
    "title": null,
    "updated": "2024-08-05T06:24:12.089000+00:00",
    "vendors": [],
    "weaknesses": []
  },
  "nvd": {
    "cpes": [
      "cpe:2.3:a:splunk:splunk:*:*:*:*:enterprise:*:*:*",
      "cpe:2.3:a:splunk:splunk:*:*:*:*:light:*:*:*"
    ],
    "created": "2018-10-23T21:31:39.257000+00:00",
    "description": "Directory traversal vulnerability in the Splunk Django App in Splunk Enterprise 6.0.x before 6.0.14, 6.1.x before 6.1.13, 6.2.x before 6.2.14, 6.3.x before 6.3.10, 6.4.x before 6.4.6, and 6.5.x before 6.5.3; and Splunk Light before 6.6.0 allows remote authenticated users to read arbitrary files via unspecified vectors.",
    "metrics": {
      "cvssV2_0": {
        "score": 4.0,
        "vector": "AV:N/AC:L/Au:S/C:P/I:N/A:N"
      },
      "cvssV3_0": {
        "score": 6.5,
        "vector": "CVSS:3.0/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:N/A:N"
      },
      "cvssV3_1": {},
      "cvssV4_0": {}
    },
    "nvd_repo_path": "2018/CVE-2018-7431.json",
    "references": [
      "https://www.splunk.com/view/SP-CAAAP5T"
    ],
    "title": null,
    "updated": "2019-01-25T15:10:24.520000+00:00",
    "vendors": [
      "splunk",
      "splunk$PRODUCT$splunk"
    ],
    "weaknesses": [
      "CWE-22"
    ]
  },
  "opencve": {
    "changes": [],
    "cpes": {
      "data": [
        "cpe:2.3:a:splunk:splunk:*:*:*:*:enterprise:*:*:*",
        "cpe:2.3:a:splunk:splunk:*:*:*:*:light:*:*:*"
      ],
      "providers": [
        "nvd"
      ]
    },
    "created": {
      "data": "2018-10-23T21:00:00+00:00",
      "provider": "mitre"
    },
    "description": {
      "data": "Directory traversal vulnerability in the Splunk Django App in Splunk Enterprise 6.0.x before 6.0.14, 6.1.x before 6.1.13, 6.2.x before 6.2.14, 6.3.x before 6.3.10, 6.4.x before 6.4.6, and 6.5.x before 6.5.3; and Splunk Light before 6.6.0 allows remote authenticated users to read arbitrary files via unspecified vectors.",
      "provider": "mitre"
    },
    "metrics": {
      "cvssV2_0": {
        "data": {
          "score": 4.0,
          "vector": "AV:N/AC:L/Au:S/C:P/I:N/A:N"
        },
        "provider": "nvd"
      },
      "cvssV3_0": {
        "data": {
          "score": 6.5,
          "vector": "CVSS:3.0/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:N/A:N"
        },
        "provider": "nvd"
      },
      "cvssV3_1": {
        "data": {},
        "provider": null
      },
      "cvssV4_0": {
        "data": {},
        "provider": null
      },
      "kev": {
        "data": {},
        "provider": null
      },
      "ssvc": {
        "data": {},
        "provider": null
      },
      "threat_severity": {
        "data": null,
        "provider": null
      }
    },
    "references": {
      "data": [
        "https://www.splunk.com/view/SP-CAAAP5T"
      ],
      "providers": [
        "mitre",
        "nvd"
      ]
    },
    "title": {
      "data": null,
      "provider": null
    },
    "updated": {
      "data": "2024-08-05T06:24:12.089000+00:00",
      "provider": "mitre"
    },
    "vendors": {
      "data": [
        "splunk",
        "splunk$PRODUCT$splunk"
      ],
      "providers": [
        "nvd"
      ]
    },
    "weaknesses": {
      "data": [
        "CWE-22"
      ],
      "providers": [
        "nvd"
      ]
    }
  }
}